packages/trackerless-network/src/logic/RandomGraphNode.ts

```diff
@@ -308,7 +311,7 @@ export class RandomGraphNode extends EventEmitter<Events> {
             markAndCheckDuplicate(this.duplicateDetectors, msg.messageId!, msg.previousMessageRef)
         }
         this.emit('message', msg)
-        this.config.propagation.feedUnseenMessage(msg, this.getPropagationTargets(msg), previousNode ?? null)
+        this.config.propagation.feedUnseenMessage(msg, this.getPropagationTargets(msg), null)
```

Comment:
Is this change part of a separate functionality change? Or how this relates to the closing temporary connections feature?